### PR TITLE
Feature/add sensor data to response

### LINKED ISSUE #192: Add sensor data to sensor response
image

Currently the sensor data is not returned on the get functions. 
Also on create etc the data is not returned

### PR DESCRIPTION
**Pull request by doriengr**
Close #192

### CODE REVIEW

**Review by Mohammad96Assaf — CHANGES_REQUESTED** [review]
The changes inside **handleMqttMessage** are causing an issue, and the data from TTN cannot be converted correctly.
Please fix the problem and verify if the data is still read from TTN and stored in the database correctly.

**Comment by doriengr** [comment]
> The changes inside **handleMqttMessage** are causing an issue, and the data from TTN cannot be converted correctly. Please fix the problem and verify if the data is still read from TTN and stored in the database correctly.

Csn you show me how to test it?

**Comment by doriengr** [comment]
> The changes inside **handleMqttMessage** are causing an issue, and the data from TTN cannot be converted correctly. Please fix the problem and verify if the data is still read from TTN and stored in the database correctly.

Is fixed 👍

File: internal/storage/postgres/sensor/entities/mqtt_payload.go [hunk 1]
```diff
@@ -7,9 +7,11 @@ type Watermark struct {
 }
 
 type MqttPayload struct {
-	DeviceID    string      `json:"device_id"`
+	Device      string      `json:"device"`
 	Battery     float64     `json:"battery"`
 	Humidity    float64     `json:"humidity"`
 	Temperature float64     `json:"temperature"`
+	Latitude    float64     `json:"latitude"`
+	Longitude   float64     `json:"longitude"`
```
Comment (Mohammad96Assaf) [inline internal/storage/postgres/sensor/entities/mqtt_payload.go:15]: Why are latitude and longitude being added here? These values are already saved once directly as sensor attributes, so there’s no need to include them again in the sensor data, every time the data is stored.

We’ve previously discussed the structure of sensor data in #233.

**Review by Mohammad96Assaf — CHANGES_REQUESTED** [review]

**Review by Mohammad96Assaf — APPROVED** [review]

**Pull request merged**